### PR TITLE
chore: breaking change in DeviceAccess

### PR DESCRIPTION
**Pull request by mhier**

### CODE REVIEW

File: src/DoocsBackend.cc [hunk 1]
```diff
@@ -176,14 +178,14 @@ namespace ChimeraTK {
       // if again error received, throw exception
       if(rc && isCommunicationError(dst.error())) {
         lk.unlock();
-        setException();
-        throw ChimeraTK::runtime_error(std::string("Cannot read from DOOCS property: ") + dst.get_string());
+        auto message = std::string("Cannot read from DOOCS property: ") + dst.get_string();
```
Comment (killenb) [inline src/DoocsBackend.cc:181]: Are you sure that dst.get_string() is giving the address/name of the property? Isn't this supposed to give the data content as a string? This did not change, though, so it's OK for this merge request.

**Review by killenb — APPROVED** [review]
There is only one commit, mixing code formatting in 40 files with minimal functional changes. These should have been separated. It is too late to disentangle it now without major effort, so let's just continue and try to improve in future.

**Comment by mhier** [comment]
> There is only one commit, mixing code formatting in 40 files with minimal functional changes. These should have been separated. It is too late to disentangle it now without major effort, so let's just continue and try to improve in future.

Yes, sorry about that. I only noticed when it was too late. This is the reason why code formatting should always be correct in the repo.

File: src/DoocsBackend.cc (hunk 1 above)
Comment (mhier) [reply src/DoocsBackend.cc:181]: I am pretty sure it works, as I likely have added this in need for this particular information.

**Pull request merged**